webpack/assets/javascripts/react_app/components/LoginPage/index.js

```diff
@@ -0,0 +1,22 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from './LoginPageActions';
+import reducer from './LoginPageReducer';
+
+import LoginPage from './LoginPage';
+
+// map state to props
+const mapStateToProps = () => ({});
+
+// map action dispatchers to props
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+
+// export reducers
+export const reducers = { loginPage: reducer };
+
+// export connected component
+export default connect(
```

Comment:
Does this component need to be connected? looks like you don't have any controllers/reducers/selectors.
Won't it be the same as `export { default } from './LoginPage';

Reply:
at the next stage we might want it to be connected and pass important information into the app by the redux store. yes, we can export the component itself right now and comment this line for a future use.

Reply:
Got it. imo removing because the redux infra code are just boilerplate and we can generate it with `npm run create-react-component` anyway.